app/models/recipe.rb

```diff
@@ -0,0 +1,3 @@
+class Recipe < ApplicationRecord
+  belongs_to :User
```

Comment:
The Recipe model should have a "has_many" association with RecipeFood model based on the ER Diagram.